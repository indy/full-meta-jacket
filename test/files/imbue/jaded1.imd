---
---
Woah! jade _and_ markdown, very **cool** we can even link to [stuff](http://google.com) markdown

Here's some more text that should be parsed as markdown
