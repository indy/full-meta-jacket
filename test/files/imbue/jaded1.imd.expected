<p>Woah! jade <em>and</em> markdown, very <strong>cool</strong> we can even link to <a href="http://google.com">stuff</a> markdown</p>

<p>Here&#39;s some more text that should be parsed as markdown</p>
